File: fixtures/scale/kotlin/large/src/cryptobench/registry/DigraphRegistry.kt
package cryptobench.registry

import cryptobench.core.CipherSuite
import cryptobench.ciphers.playfair.PlayfairSuite
import cryptobench.ciphers.twosquare.TwoSquareSuite
import cryptobench.ciphers.foursquare.FourSquareSuite
import cryptobench.ciphers.hill.HillSuite
import cryptobench.ciphers.bifid.BifidSuite
import cryptobench.ciphers.adfgvx.AdfgvxSuite

object DigraphRegistry {
    fun suites(): List<CipherSuite> = listOf(
        PlayfairSuite(),
        TwoSquareSuite(),
        FourSquareSuite(),
        HillSuite(),
        BifidSuite(),
        AdfgvxSuite(),
    )
}

File: fixtures/scale/dart/large/lib/registry/digraph_registry.dart
import '../core/cipher_suite.dart';
import '../ciphers/playfair/playfair_suite.dart';
import '../ciphers/twosquare/twosquare_suite.dart';
import '../ciphers/foursquare/foursquare_suite.dart';
import '../ciphers/hill/hill_suite.dart';
import '../ciphers/bifid/bifid_suite.dart';
import '../ciphers/adfgvx/adfgvx_suite.dart';

List<CipherSuite> digraphSuites() => [
  PlayfairSuite(),
  TwoSquareSuite(),
  FourSquareSuite(),
  HillSuite(),
  BifidSuite(),
  AdfgvxSuite(),
];
